src/gateway/shard.rs

```diff
@@ -522,8 +521,8 @@ impl Shard {
 
     /// Shuts down the receiver by attempting to cleanly close the
     /// connection.
-    #[doc(hidden)]
-    pub fn shutdown_clean(client: &mut WsClient) -> Result<()> {
+    #[allow(dead_code)]
+    pub(crate) fn shutdown_clean(client: &mut WsClient) -> Result<()> {
```

Comment:
The websocket client is now available to the user, they should be able to call this if they'd like -- the `#[doc(hidden)]` can be removed.